rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
Yeah, but apparently that's how the signature is suppose to look like to be compatible with other clients.
So it's a signature of some data with prepended text and definitely not a transaction hash as the change you did suggests. (`Signature -> PostSignTransaction`)